relay-quotas/src/redis.rs

```diff
@@ -99,18 +110,26 @@ impl<'a> RedisQuota<'a> {
         UnixTimestamp::from_secs(next_start)
     }
 
-    fn key(&self) -> String {
+    /// Returns the key of the quota.
+    pub fn key(&self) -> String {
         // The subscope id is only formatted into the key if the quota is not organization-scoped.
         // The organization id is always included.
         let subscope = match self.quota.scope {
             QuotaScope::Organization => None,
             scope => self.scoping.scope_id(scope),
         };
 
+        // 0 is arbitrary, we just need to ensure global quotas from different orgs have the same key.
+        let org = if self.quota.scope == QuotaScope::Global {
```

Comment:
We need this logic everywhere we access `self.scoping.organization_id` -> in `shift`